imports/ui/QuestionListItem.jsx

```diff
@@ -30,6 +32,7 @@ export class QuestionListItem extends ListItem {
 
   render () {
     const s = this.props.session
+    const c = Courses.findOne({ _id: this.props.question.courseId })
```

Comment:
We don't want to query the database inside of a React component. What I think we want here will be a filter for the course that the user has chosen (based on a dropdown in page_container, as we discussed). We should chat a bit about the refactor for the page_container so that a user's state on the app has a course in it.

Reply:
The purpose of querying the database in this instance is to display the course code on the question cards so that the user can easily identify what course a question is in. Perhaps this could be done through the props? However if the page_container is refactored then this may all change.